pages/general/adopting.md

```diff
@@ -2,15 +2,15 @@
 
 By adopting the guidelines from the API Enhancement Proposals, you establish a much 
 tighter and efficient possibility space. APIs that follow the 
-[AEP guidance](https://aep.dev) from the design stage are:
+[AEP guidance](https://aep.dev) from the design stage:
 
-- More consistent APIs within and across teams, reducing cognitive load for consumers
-- Argue less about API design decisions, saving time thanks to AEPs explar design patterns
+- are more consistent  within and across teams, reducing cognitive load for API consumers
+- reduce arguments  about API design decisions, thanks to the AEP specification, tools, and exemplar design patterns
 
 Having an AEP-compliant API also means benefitting from the AEP ecosystem of tooling, such as:
 
 - Linters and validators to ensure adherence to the AEP design specification [proto]()/[openapi]()
-- Command line tool that make it easier to work with APIs ([aepcli](https://github.com/aep-dev/aepcli))
+- A command line tool interface genreator that make it easier to work with APIs ([aepcli](https://github.com/aep-dev/aepcli))
```

Comment:
```suggestion
- A dynamic command line interface generator that make it easier to work with APIs ([aepcli](https://github.com/aep-dev/aepcli))
```

I think "dynamic" is important here - it implies it get change without code changes (e.g. parsing and generating static code that has to be distributed and released.